src/Raven.Server/Rachis/RachisConsensus.cs

```diff
@@ -440,6 +444,9 @@ public unsafe void Initialize(StorageEnvironment env, RavenConfiguration configu
                 DebuggerAttachedTimeout.LongTimespanIfDebugging(ref _electionTimeout);
 
                 ContextPool = new ClusterContextPool(changes, _persistentState, configuration.Memory.MaxContextSizeToKeep);
+                TxMerger = new ClusterTransactionOperationsMerger(this, configuration, time, shutdown);
```

Comment:
we have some "local" operations that are running on the server store, they are also going through here (local cert, for example)

Reply:
This will be problematic, since they are not using ClusterOperationContext, yet same store. I've reviewed them and most of those operations are ultra-rare. Probably we do not need to apply tx merger there. Now I'm focusing only on all of the operations bound to ClusterOperationContext.